tests/app/dao/notification_dao/test_notification_dao.py

```diff
@@ -1131,6 +1131,49 @@ def test_dao_get_notifications_by_to_field_escapes(
     )) == expected_result_count
 
 
+@pytest.mark.parametrize('search_term, expected_result_count', [
+    ('foobar', 1),
+    ('foo', 2),
+    ('bar', 2),
+    ('foo%', 1),
+    ('%%bar', 1),
+    ('%_', 1),
+    ('%', 2),
+    ('_', 1),
+    ('/', 1),
+    ('\\', 1),
+    ('baz\\baz', 1),
+    ('%foo', 0),
+    ('%_%', 0),
+    ('test@example.com', 5),
+])
+def test_dao_get_notifications_by_to_field_escapes_reference(
```

Comment:
what does it mean that it *escapes* reference?

Reply:
`%` and `_` are wildcard characters in SQL, but we want them to be interpreted literally. There’s a bit more detail in the original search by recipient implementation: bdd77f9150fde59d71fdc837ff4cd2cbf0d62896

Reply:
ahh, I see, thanks 💖 
Maybe we could change test name to `test_dao_get_notifications_by_to_field_escapes_special_chars_for_reference` ?
What do you think :)